### PR TITLE
Add basic Travis CI build

### PR DESCRIPTION
**Pull request by jfbastien**
No setup or tests yet, I first want to see the basics working properly.

### CODE REVIEW

**Comment by kripken** [comment]
lgtm, but lol, it reports failing checks right now on this pull, even without doing anything?

File: README.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ Binaryen is a compiler and toolchain infrastructure library for WebAssembly, wri
 
 To provide those capabilities, Binaryen has a simple and flexible API for **representing and processing** WebAssembly modules. The interpreter, validator, pretty-printer, etc. are built on that foundation. The core of this is in [wasm.h](https://github.com/WebAssembly/binaryen/blob/master/src/wasm.h), which contains classes that define a WebAssembly module, and tools to process those. For a simple example of how to use Binaryen, see [test/example/find_div0s.cpp](https://github.com/WebAssembly/binaryen/blob/master/test/example/find_div0s.cpp), which creates a module and then searches it for a specific pattern.
 
+Consult the [contributing instructions](Contributing.md) if you're interested in participating.
+
+Current build status: [![Build Status](https://travis-ci.org/WebAssembly/binaryen.svg?branch=master)](https://travis-ci.org/WebAssembly/binaryen)
```
Comment (kripken) [inline README.md]: this might be higher up perhaps?

File: README.md [hunk 2]
```diff
@@ -30,9 +34,9 @@ $ ./build.sh
 ```
 or
 ```
-cmake && make
+cmake ./ && make
```
Comment (kripken) [inline README.md]: what does `./` do here? is it the same as `.`?

Comment (jfbastien) [inline README.md]: Yes.

File: README.md (hunk 1 above)
Comment (jfbastien) [inline README.md]: I can do it if you want, but it seems like having the intro first is friendlier?

Comment (kripken) [inline README.md]: no need, seems ok either way

File: README.md (hunk 2 above)
Comment (kripken) [inline README.md]: ok

**Comment by kripken** [comment]
lgtm squashed

File: README.md (hunk 2 above)
Comment (jfbastien) [inline README.md]: Changed to .

**Pull request merged**